src/dmd/globals.d

```diff
@@ -370,7 +371,10 @@ extern (C++) struct Global
 
     extern (C++) void _init()
     {
+        // add NULL for the backend, but retain _version as dstring for the frontend
         _version = import("VERSION") ~ '\0';
+        _version = _version[0 .. $-1];
+        _versionNumber = calcVersionNumber();
```

Comment:
Not sure about this change, this is now eagerly evaluated and not on first use (if any).

Reply:
I don't think that it matters, but I moved the calculation to compile-time.